### ISSUE TITLE
Is there permission to customize this launcher and use it for a different game?

### ISSUE THREAD

**Issue by IGNRexI** [neon-nyan/Collapse#73]
Is there permission to customize this launcher and use it for a different game?
I want to use this as a launcher for a different game, it's so cool. I couldn't find the games in the game class where?
![image](https://user-images.githubusercontent.com/36341134/212259714-6654b69c-bd98-4770-b661-52ca79763390.png)
I want to delete the games here and add a different game.

**Comment by Cryotechnic** [comment]
There are currently no ways to add support for custom games to the launcher due to the way it was designed and implemented. While I do think that it would be possible in theory, doing so would require a (somewhat) major rewrite of the backend infrastructure to enable this kind of action. 

Tldr; not out of the question but there are no current plans to do so.

**Comment by neon-nyan** [comment]
As mentioned by @Cryotechnic (thank you for helping me to answer this issue XD)

Firstly, this launcher was meant to be a launcher for "miHoYo PC Games" only and unfortunately, we are currently have no plan to add another games.

And for your information that the list of the games supported are included inside of this metadata from ``CollapseLauncher-ReleaseRepo`` repository:
https://github.com/neon-nyan/CollapseLauncher-ReleaseRepo/blob/main/metadata/metadatav2_stableconfig.json

So yes, unfortunately. There's currently no way to add another games other than miHoYo's PC Games that's currently available right now (Honkai Impact 3rd and Genshin Impact).

But, we will be going to add another miHoYo's game once they are released into the public (Honkai: Star Rail and Zenless Zone Zero)
